javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/DeclarativeConfigPropertiesBridge.java

```diff
@@ -46,26 +49,86 @@
  *         string_key: value
  * </pre>
  */
-final class DeclarativeConfigPropertiesBridge implements ConfigProperties {
+public final class DeclarativeConfigPropertiesBridge implements ConfigProperties {
```

Comment:
should this class be part of the public API?

Reply:
this class will be needed for the spring starter - but we can create a factory method